docs/schema.md

```diff
@@ -117,12 +119,12 @@ All properties marked as "not undefined" will be present in the end record, the
   };
 
   /**
-   * Current heading
+   * The current heading
    */
   currentHierarchy?: string;
 
   /**
-   * The size of the content `content` property, in bytes
+   * The size of the `content` property (in bytes)
    */
   contentLength?: number;
 
```

Comment:
Why don't we document the `recordUrl` and `position`?

Reply:
I was not sure what it was ahah